utils/release_notes.rst.in

```diff
@@ -0,0 +1,30 @@
+==================================
+Cyclus @VERSION@ Release Notes
+==================================
+:Author: Cyclus Developers
+:Website: http://fuelcycle.org/
+:Contact: https://groups.google.com/forum/#!forum/cyclus-dev
+
+The Cyclus core team is excited to announce the latest stable version of the
+cyclus ecosystem!
+
+Major features
+==============
+!--- TODO: Fill in major features --!
```

Comment:
done

On Wed, May 6, 2015 at 11:31 AM, Anthony Scopatz <notifications@github.com>
wrote:

> In utils/release_notes.rst.in
> <https://github.com/cyclus/ciclus/pull/63#discussion_r29775947>:
>
> > @@ -0,0 +1,30 @@
> > +==================================
> > +Cyclus @VERSION@ Release Notes
> > +==================================
> > +:Author: Cyclus Developers
> > +:Website: http://fuelcycle.org/
> > +:Contact: https://groups.google.com/forum/#!forum/cyclus-dev
> > +
> > +The Cyclus core team is excited to announce the latest stable version of the
> > +cyclus ecosystem!
> > +
> > +Major features
> > +==============
> > +!--- TODO: Fill in major features --!
>
> Exactly. These lines should never be rendered. If you want something to be
> rendered, I think a warning directive would be much better / flashier.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/63/files#r29775947>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192